### PR TITLE
Fix CI, with Python 3.6 support.

### LINKED ISSUE #642: Tox>=4 passenv should be comma separated
https://github.com/jazzband/django-redis/actions/runs/3862393642/jobs/6583934580

```
py37-dj22-redislatest: failed with pass_env values cannot contain whitespace, use comma to have multiple values in a single line, invalid values found 'CI GITHUB*'
  py37-dj22-redislatest: FAIL code 1 (0.01 seconds)
  evaluation failed :( (0.56 seconds)
```

https://github.com/tox-dev/tox/issues/2658

### PR DESCRIPTION
**Pull request by iurisilvio**
Fix #642.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
       Django ${{ matrix.django-version }},
       Redis.py ${{ matrix.redis-version }}
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (WisdomPill) [inline .github/workflows/ci.yml:12]: why not using ubuntu-latest?

**Review by WisdomPill — APPROVED** [review]
LGTM!

just one question on ubuntu version

File: .github/workflows/ci.yml (hunk 1 above)
Comment (iurisilvio) [reply .github/workflows/ci.yml:12]: Latest don't support Python 3.6.

I hit the issue here: https://github.com/jazzband/django-redis/actions/runs/3859407256/jobs/6578888882#step:3:7

Comment (iurisilvio) [reply .github/workflows/ci.yml:12]: I think it can be 22.10, I'll try it. Latest is 23.04.

Comment (WisdomPill) [reply .github/workflows/ci.yml]: already replied in [here](https://github.com/jazzband/django-redis/pull/644#issuecomment-1387068318)

Comment (iurisilvio) [reply .github/workflows/ci.yml:12]: I tried 22.04 and it fails too.

I agree it could be setup with pyenv, I'll try it in another PR.

**Pull request merged**